src/layouts/BaseLayout.js

```diff
@@ -1,16 +1,17 @@
-import React from "react";
-import { Link } from "react-router-dom";
 import { Container, Header, Menu } from "semantic-ui-react";
 
+import { Link } from "react-router-dom";
+import React from "react";
+
 function BaseLayout(props) {
   return (
     <div>
       <Menu fixed="top" inverted>
         <Container>
-          <Menu.Item as="a" header>
```

Comment:
Was throwing an error ( this actually generates an a link, so does Link, which ends up having two embedded <a><a> )